config.py-example

```diff
@@ -15,7 +15,8 @@ DATASOURCES = {
 							"boards": "*",
 		},
     "telegram": {},
-    "twitterv2": {}
+    "twitterv2": {},
+    "url_scraper": {},
```

Comment:
I'll go ahead and remove it from the defaults since you are concerned, but if a student ran the datasource under that impression, they would, instead of comments/posts, receive HTML code of the facebook login page. That ought to become quickly apparent if it was not already clear in the datasource description. I'm not sure of the real downside to this, but maybe they would email you or open an issue which is time consuming to respond to.

I think an actual issue that would create confusing might be in using the various processors. Some might fail or return strange results as they are designed with text in mind. I already thought it would be worth upgrading this to extract the text (which more naturally fix the `body` field) and pushing the HTML code to a `source_code` field. That just wasn't immediately necessary for my intent.

It may also be worth noting that the defaults only matter to users that install 4CAT directly to their machine or server. If I found 4CAT on Github or from a blog post, I would probably assume that all working features would be readily available and not rely on me searching the source code or Github wiki to activate them. Perhaps a semi-solution might be to add all the datasources to the config with an `active` attribute as True and False. That would couple nicely if we move to a more interactive control panel.

Reply:
Yep, that's a good QoL suggestion, and aids discoverability of data sources. And then later there could be a page in the interface where you simply flip switches for specific data sources.

RE: Processors - I wonder if we should explicitly have two categories of 4CAT data sources - social media (i.e. following the thread/post/subject format) and other. I think most processors would mostly be useful to only one of the two, or need explitly defined alternate code paths to deal with both. It would also make it easier to set the right expectations in the interface. It puts this data source sort of beyond the original scope of 4CAT though, so I'm not sure how to best handle that. Maybe we can discuss it in a meeting sometime soon with Sal.

RE: Users would notice if the output is not what they expect - true, but they'd only see that *after* Selenium has spent an hour fetching Facebook pages ;)